metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -109,6 +114,15 @@ public Long value() {
                 return time.milliseconds() - lastAppliedRecordTimestamp();
             }
         }));
+
+        if (dualWriteModeState.get()) {
+            registry.ifPresent(r -> r.newGauge(ZOO_KEEPER_WRITE_BEHIND_LAG, new Gauge<Long>() {
+                @Override
+                public Long value() {
+                    return lastCommittedRecordOffset() - dualWriteOffset();
```

Comment:
Since we won't start updating dualWriteOffset until the migration is done, this returned value will just be equal to the lastCommittedRecordOffset which is ever-increasing. 

I think if dualWriteOffset is zero, we should short-circuit the lag calculation and just return zero. WDYT?